tests/by-util/test_ls.rs

```diff
@@ -3117,3 +3117,12 @@ fn test_dereference_symlink_file_color() {
         .succeeds()
         .stdout_is(out_exp);
 }
+
+#[test]
+fn test_tabsize_option() {
+    let scene = TestScenario::new(util_name!());
+
+    scene.ucmd().args(&["-T", "3"]).succeeds();
```

Comment:
I would also like to see a new test with what it should actually return once it is implemented
and marked as ignored.
Could you please do that ? thanks

Reply:
Ok, done.

Reply:
Spellcheck doesn’t like “aaaaaaaa”, “bbbb”, etc. Should I just add them to the spell ignore?

Reply:
yeah